lib/App.php

```diff
@@ -53,6 +57,8 @@ public function setOutput(OutputInterface $output): void {
 	public function notify(INotification $notification): void {
 		$notificationId = $this->handler->add($notification);
 
+		$this->mailNotifications->sendEmails($notification->getUser(), true);
```

Comment:
this will have quite a negative impact on performance.
Maybe we an do it similar to the push notifications and just "queue" the sending when the "defer" of notifications is on and send them at the end of the request.